src/views/Dashboard/Customer/Home/RequestTest/PaymentReceiptDialog.vue

```diff
@@ -203,10 +229,35 @@ export default {
       this.txWeight = this.web3.utils.fromWei(String(txWeight.partialFee), "ether")
     },
 
+    async checkMetamask(){
+      this.metamask = await startApp()
+      this.role = process.env.VUE_APP_ROLE
```

Comment:
I think you don't need to store the ENV inside the data bc it is just one time use

Reply:
removed. thanks mas @bukandicki. please kindly re-review